sites/svelte.dev/content/blog/2023-03-01-whats-new-in-svelte-march-2023.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "What's new in Svelte: March 2023"
+description: "SvelteHack, post-1.0 SvelteKit improvements and a huge showcase"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+March is coming in hot with a number of essential features having been added to SvelteKit following its 1.0 launch. Check out [last week's blog post](https://svelte.dev/blog/streaming-snapshots-sveltekit) for more details.
+
+We're also excited to have launched [SvelteHack](https://hack.sveltesociety.dev/) last month! Calling all Svelte developers, new and old, to make something awesome before the April 17th deadline.
```

Comment:
```suggestion
We're also excited to have launched [SvelteHack](https://hack.sveltesociety.dev/) last month with over $12,000 in prizes! Calling all Svelte developers, new and old, to make something awesome before the April 17th deadline.
```

Reply:
Is it "we" that have launched this? Should we clarify that it's Svelte Society?